src/zb_links/db/init_tables/seed_db.py

```diff
@@ -122,11 +113,19 @@ def seed_author_ids():
     connection.execute(data_row)
 
 
-@seedbp.cli.command("all")
-@click.pass_context
-def click_seed_all(ctx):
-    seed_partner.invoke(ctx)
-    seed_source.invoke(ctx)
-    seed_target.invoke(ctx)
-    seed_link.invoke(ctx)
-    seed_author_ids.invoke(ctx)
+# @seedbp.cli.command("all")
```

Comment:
Delete it. The old code remains in the git history.

Reply:
done